### PR TITLE
Remove sdx-common clone in docker

### PR DESCRIPTION
**Pull request by gemmai95**
Remove sdx-common clone in docker.

## Checklist
  - [x] CHANGELOG.md updated? (if required)

### CODE REVIEW

**Comment by gemmai95** [comment]
[(3) Remove sdx common clone in Dockerfiles](https://trello.com/c/AfHeYs4M/400-3-remove-sdx-common-clone-in-dockerfiles)

File: Dockerfile [hunk 1]
```diff
@@ -5,8 +5,8 @@ RUN apt-get upgrade -y
 RUN apt-get install -yq git gcc make build-essential python3-dev python3-reportlab
 
 RUN mkdir -p /app/logs
-RUN git clone -b 0.7.0 https://github.com/ONSdigital/sdx-common.git
-RUN pip3 install ./sdx-common
+
+RUN pip3 install --no-cache-dir -U -r requirements.txt
```
Comment (necrophonic) [inline Dockerfile]: Doesn't actually run if you build the docker container :)

This line is installing the `requirements.txt` that's supplied by the base container; not the `requirements.txt` of the app - the `requirements.txt` needs to be copied in.

However, even with that it doesn't seem to be compiling?

Comment (necrophonic) [inline Dockerfile]: You can simplify some of the steps by just copying in and running the `Makefile`

File: Dockerfile [hunk 2]
```diff
@@ -5,10 +5,11 @@ RUN apt-get upgrade -y
...
 
 RUN mkdir -p /app/logs
-RUN git clone -b 0.7.0 https://github.com/ONSdigital/sdx-common.git
-RUN pip3 install ./sdx-common
 
 COPY app /app
 COPY startup.sh /startup.sh
+COPY Makefile /Makefile
+
+CMD make build
```
Comment (necrophonic) [inline Dockerfile]: Missing the correct `requirements.txt`

**Comment by gemmai95** [comment]
[(3) Remove sdx common clone in Dockerfiles](https://trello.com/c/AfHeYs4M/400-3-remove-sdx-common-clone-in-dockerfiles)

**Pull request merged**